internal/cli/guided_init.go

```
@@ -316,6 +340,23 @@ func promptProfile(reader *bufio.Reader, output io.Writer, defaultValue activati
+		value, err := promptValue(reader, output, "Evidence profile (github-delivery/v1, multica-execution/v1)", string(defaultValue))
+		if err != nil {
+			return "", err
+		}
+		profile, err := domain.ParseEvidenceProfile(value)
+		if err == nil {
+			return profile, nil
+		}
+		last = err
+		fmt.Fprintf(output, "Invalid evidence profile. Use github-delivery/v1 or multica-execution/v1 (%d attempt(s) left).\n", guidedPromptAttempts-attempt)
+	}
+	return "", fmt.Errorf("%v. Next: rerun guided init and choose an explicit evidence profile; no changes were made.", last)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)